.github/workflows/check.yml

```diff
@@ -118,9 +121,25 @@ jobs:
           fi
           for sanitizer in $SANITIZERS; do
             echo "Running tests with $sanitizer sanitizer..."
+            # shellcheck disable=SC2078
+            if [ "${{ env.DUMP_SIMULATION_SEEDS }}" ]; then
+              export DUMP_SIMULATION_SEEDS="${{ env.DUMP_SIMULATION_SEEDS }}/sanitizer/$sanitizer"
+            fi
             RUSTFLAGS="-Z sanitizer=$sanitizer" RUSTDOCFLAGS="-Z sanitizer=$sanitizer" cargo +nightly nextest run -Z build-std --features ci --target "$TARGET"
           done
 
+      - name: Print simulation seeds
+        if: env.DUMP_SIMULATION_SEEDS
+        run: find ${{ env.DUMP_SIMULATION_SEEDS }}
+
```

Comment:
Presumably you don't need this...

```suggestion
```

Reply:
Not strictly, since we also provide them as artifacts.